helpers/pyband/pyband/message.py

```diff
@@ -113,3 +113,187 @@ def validate(self) -> bool:
         self.amount.validate()
 
         return True
+
+
+@dataclass
+class MsgCreateDataSource(Msg):
+    owner: Address
+    name: str
+    description: str
+    script: str
```

Comment:
I think we can should receive bytes (or add another field in case you want to use destination file path)